Tangent_Mode.py

```diff
@@ -0,0 +1,44 @@
+import findspark
```

Comment:
Do we need Spark for Tangent_Mode? :P

Reply:
Something I noticed right after making this PR :'(